src/video/src/Video_Desktop.cpp

```diff
@@ -59,7 +90,7 @@ bool Video::setup_gl()
     _platform_specific->window = SDL_CreateWindow("SpelunkyPSP",
                               SDL_WINDOWPOS_UNDEFINED,
                               SDL_WINDOWPOS_UNDEFINED,
-                              480 * 2, 272 * 2,
+                              480 * 4, 272 * 4,
```

Comment:
Query for screen size and set a window with a fixed-ratio, i.e `0.8 * screen_height`.